src/app/static/src/app/actions.ts

```diff
@@ -30,6 +31,15 @@ const currentRegionBaseline = (state: RootState) => {
 
 export const actions: ActionTree<RootState, RootState> = {
 
+    async [RootAction.SetCurrentRegionBaselineSettings](context, payload: DynamicFormData) {
+        const {dispatch, commit} = context;
+        commit(RootMutation.SetCurrentRegionBaselineSettings, payload);
```

Comment:
we were previously just committing the settings to the store, then fetching data on "next". but now, if the baseline settings are open while you're viewing figures, we need to fetch fresh data in real time as settings are changed, hence this new action